// @ts-check
import process from "process";

process.on("unhandledRejection", (reason, p) => {
  console.error("ðŸ’¥ Unhandled Rejection at:", p, "reason:", reason);
});
process.on("uncaughtException", (error) => {
  console.error(`ðŸ’¥ Caught exception: ${error}`);
  console.error(`ðŸ’¥ Exception origin: ${error.stack}`);
});
